core/src/main/java/hudson/PluginManager.java

```diff
@@ -2409,8 +2409,15 @@ public UberClassLoader(List<PluginWrapper> activePlugins) {
 
         @Override
         protected Class<?> findClass(String name) throws ClassNotFoundException {
-            if (name.startsWith("SimpleTemplateScript")) { // cf. groovy.text.SimpleTemplateEngine
-                throw new ClassNotFoundException("ignoring " + name);
+            String[] namePrefixesToSkip = new String[] {
```

Comment:
This method might be called quite a bit, so better to make this into a constant.